packages/react/src/executors/module-federation-ssr-dev-server/module-federation-ssr-dev-server.impl.ts

```diff
@@ -76,9 +76,14 @@ function startSsrStaticRemotesFileServer(
   ssrStaticRemotesConfig: StaticRemotesConfig,
   context: ExecutorContext,
   options: ModuleFederationSsrDevServerOptions
-) {
+):
+  | AsyncGenerator<{ success: boolean; baseUrl?: string }>
+  | AsyncIterable<{ success: boolean; baseUrl?: string }> {
   if (ssrStaticRemotesConfig.remotes.length === 0) {
-    return;
+    return createAsyncIterable(({ next, done }) => {
```

Comment:
This could be simplified if `startSsrStaticRemotesFileServer` is an async generator.

```
if (ssrStaticRemotesConfig.remotes.length === 0) {
  yield { success: true};
  return;
}
```

But then `return staticRemotesIter` needs to be `yield* staticRemotesIter`.